blackbox/docs/general/builtins/aggregation.txt

```diff
@@ -488,6 +488,51 @@ faster than on strings.  The advantage is that the ``arbitrary`` function does
 very little to no computation as for example ``max`` aggregation function would
 do.
 
+.. _aggregation-hll-distinct:
+
+``hyperloglog_distinct``
+========================
+
+.. note::
+
+   The ``hyperloglog_distinct`` aggregate function is an :ref:`enterprise
+   feature <enterprise_features>`.
+
+The ``hyperloglog_distinct`` aggregate function calculates an approximate count
+of distinct non-null values using the `HyperLogLog++`_ algorithm.
+
+The return value data type is always a long.
+
+The first argument can be a reference to a column of all
+:ref:`sql_ddl_datatypes_primitives`, :ref:`sql_ddl_datatypes_compound` and
```

Comment:
I read this first as if the compound types work as well - because it's written as enumeration (`,`). I think there should be a dot instead.